src/docs/implementations/dynatrace.adoc

```diff
@@ -4,25 +4,170 @@ Jon Schneider <jschneider@pivotal.io>
 ----
 DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    public String uri() {
+        // Read the URI from the environment variables or a config file
+        // Should contain the path, e. g.: https://your-environment-id.live.dynatrace.com/api/v2/metrics/ingest
+        return MY_DYNATRACE_URI;
+    }
+
     @Override
     public String apiToken() {
+        // Read from the environment or a config file.
+        // We do *not* recommend hardcoding token or URI here.
         return MY_TOKEN;
     }
 
+    @Override
+    @Nullable
+    public String get(String k) {
+        return null;
+    }
+};
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+----
+
+If you are working with Spring Boot, you can instead put the following properties in your `application-dynatrace.yml` in the `resources` folder.
+It is possible to reference environment variables using the Spring property placeholder notation here.
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    enabled: true
+
+    api-version: v2
+    # e. g. https://your-environment-id.live.dynatrace.com/api/v2/metrics/ingest
+    uri: ${ENV_VAR_DT_INGEST_URI}   
+    api-token: ${ENV_VAR_DT_INGEST_TOKEN}
+----
+
+=== API v2 [[bookmark-apiv2]]
+
+When the API version is configured to "v2", the registry will send data using the https://www.dynatrace.com/support/help/dynatrace-api/environment-api/metric-v2/[Metrics API v2].
+The v2 exporter does not require any properties to be set except the version.
+In this case, metrics will be exported to the local OneAgent endpoint (see <<bookmark-oneagent, below>>).
+
+To use the v2 API, overwrite the `apiVersion` property:
+
+[source,java]
+----
+DynatraceConfig dynatraceConfig = new DynatraceConfig() {
+    @Override
+    public DynatraceApiVersion apiVersion() {
+        return DynatraceApiVersion.V2;
+    }
+
+    @Override
+    @Nullable
+    public String get(String k) {
+        return null;
+    }
+};
+MeterRegistry registry = new DynatraceMeterRegistry(dynatraceConfig, Clock.SYSTEM);
+----
+
+Micrometer's Spring Boot support can also bind the `api-version` property to the config automatically:
+
+[source,yml]
+----
+management.metrics.export.dynatrace:
+    api-version: v2
+----
+
+[[bookmark-oneagent]]*Minimal configuration with a local OneAgent*
+
+In the minimal configuration shown above (no URI or API token), the v2 exporter will attempt to export to the https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/[local OneAgent metrics ingest endpoint].
+Note that this only works if the a OneAgent is running on the machine and https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/#enable-the-oneagent-metric-api[the local metric API feature is enabled].
```

Comment:
```suggestion
Note that this only works if the a OneAgent is running on the host and the https://www.dynatrace.com/support/help/how-to-use-dynatrace/metrics/metric-ingestion/ingestion-methods/local-api/#enable-the-oneagent-metric-api[local OneAgent Metric API] is available.
```
With this wording we don't have to update it if this was on by default or not configurable in future.

Reply:
Nice catch